src/main/java/calculator/DelimiterParser.java

```diff
@@ -0,0 +1,54 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DelimiterParser {
+
+    private static final String CUSTOM_DELIMITER_PREFIX = "//"; // 커스텀 구분자 접두사
+    private static final String CUSTOM_DELIMITER_SUFFIX = "\\n"; // 커스텀 구분자 접미사
```

Comment:
상수화 👍👍👍
상수 이름을 그대로 설명만 하는 주석이라서, 없어도 좋아 보입니다!
```suggestion
    private static final String CUSTOM_DELIMITER_PREFIX = "//";
    private static final String CUSTOM_DELIMITER_SUFFIX = "\\n";
```